### PR TITLE
T21: Add ScrollView for signIn and SignUp page XML

### PR DESCRIPTION
**Pull request by Animansoubi**
Add ScrollView For SignUp and SignIn pages

### CODE REVIEW

File: app/src/main/res/layout/frg_sign_in.xml [hunk 1]
```diff
@@ -1,79 +1,86 @@
...
-        app:layout_constraintTop_toBottomOf="@id/input_layout_email_text"
-        app:startIconDrawable="@drawable/ic_pass">
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/et_mail"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@drawable/custom_edit_text"
+                android:hint="@string/email"
+                android:inputType="textEmailAddress"
+                android:padding="15dp" />
```
Comment (idaosk) [inline app/src/main/res/layout/frg_sign_in.xml:29]: let's not use literals in xml file. instead take this to dimen.xml please

File: app/src/main/res/layout/frg_sign_in.xml [hunk 2]
```diff
@@ -1,79 +1,86 @@
...
 
-    </com.google.android.material.textfield.TextInputLayout>
+            <com.google.android.material.textfield.TextInputEditText
+                android:id="@+id/et_pass"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@drawable/custom_edit_text"
+                android:hint="@string/password"
+                android:inputType="textPassword"
+                android:padding="15dp" />
```
Comment (idaosk) [inline app/src/main/res/layout/frg_sign_in.xml:50]: same here

File: app/src/main/res/layout/frg_sign_in.xml [hunk 3]
```diff
@@ -1,79 +1,86 @@
...
-        android:textColor="@color/black"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/input_layout_pass_text" />
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guideline_start"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:orientation="horizontal"
+            app:layout_constraintGuide_begin="250dp" />
```
Comment (idaosk) [inline app/src/main/res/layout/frg_sign_in.xml:59]: here

File: app/src/main/res/layout/frg_sign_in.xml [hunk 4]
```diff
@@ -1,79 +1,86 @@
...
+
+        <Button
+            android:id="@+id/btn_sign_in"
+            style="@style/buttonStyle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@drawable/custom_button"
+            android:padding="15dp"
```
Comment (idaosk) [inline app/src/main/res/layout/frg_sign_in.xml:67]: here

File: app/src/main/res/layout/frg_sign_in.xml [hunk 5]
```diff
@@ -1,79 +1,86 @@
...
+            android:text="@string/signIn_button_text"
+            android:textColor="@color/black"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/input_layout_pass_text" />
+
+        <ImageView
+            android:id="@+id/img_logo"
+            android:layout_width="250dp"
+            android:layout_height="250dp"
```
Comment (idaosk) [inline app/src/main/res/layout/frg_sign_in.xml:77]: here

**Review by idaosk — COMMENTED** [review]
looks great! thank you

**Review by moyarsuccess — APPROVED** [review]
Approve with some minor changes

**Pull request merged**